### DISCUSSION TITLE
Optimizing Search Time

### DISCUSSION THREAD

**Discussion by Yang1032** [milvus-io/milvus#30395] [Q&A and General discussion]
Hello.
 I've been testing the search time for 500,000 vectors (1024 dimensions) on Milvus v2.3.5. 
I've experimented with HNSW parameters (M=32/64/128/256/512, efConstruction=32/64/128/512), adjusted the segment size (512MB/2048MB), and ensured flush operations. 
However, despite these changes, my search time remains constant at 400ms. What could be the issue, and what steps can I take to reduce the search time?

Additionally, when testing with 260,000 vector data, the search time shows no improvement.

**Comment by yhmo** [comment]
I guess your consistency_level is "Strong". To understand consistency_level, read this doc: https://milvus.io/docs/consistency.md#Consistency-levels

You can specify consistency_level like this:

```
results = collection.search(
     data=target_vectors,
     anns_field=VECTOR_FIELD_NAME,
     param=search_params,
     limit=topk,
     consistency_level="Bounded",
)
```

**Reply by Yang1032** [reply]
Thanks! I'll take a look at this info  and test.
